applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -363,44 +423,56 @@ static int initialize(le_audio_receive_cb recv_cb)
 			/* Channel is not assigned yet: use default */
 			channel = AUDIO_CHANNEL_DEFAULT;
 		}
+
+		for (size_t i = 0; i < ARRAY_SIZE(caps); i++) {
+			ret = bt_audio_capability_register(&caps[i]);
+			if (ret) {
+				LOG_ERR("Capability register failed");
+				return ret;
+			}
+		}
 		if (channel == AUDIO_CH_L) {
 			ret = bt_audio_capability_set_location(BT_AUDIO_DIR_SINK,
 							       BT_AUDIO_LOCATION_FRONT_LEFT);
+			if (ret) {
+				LOG_ERR("Location set failed");
+				return ret;
+			}
+
 		} else {
 			ret = bt_audio_capability_set_location(BT_AUDIO_DIR_SINK,
 							       BT_AUDIO_LOCATION_FRONT_RIGHT);
+			if (ret) {
+				LOG_ERR("Location set failed");
+				return ret;
+			}
 		}
+
+#if CONFIG_STREAM_BIDIRECTIONAL
+		ret = bt_audio_capability_set_location(BT_AUDIO_DIR_SOURCE,
+						       BT_AUDIO_LOCATION_FRONT_LEFT);
 		if (ret) {
 			LOG_ERR("Location set failed");
 			return ret;
 		}
-		ret = bt_audio_capability_register(&caps);
-		if (ret) {
-			LOG_ERR("Capability register failed");
-			return ret;
-		}
-
-		ret = bt_audio_capability_set_available_contexts(
```

Comment:
Just found that we missed the available context.
We need to expose the available context for both audio sink and source now.

Reply:
Are you sure we need to set it specifically? Since we have the BT_PACS_SRC_CONTEXT and BT_PACS_SNK_CONTEXT in Kconfig. Maybe @Thalley knows?